projects/igniteui-angular/src/lib/grid/cell.component.ts

```diff
@@ -404,6 +412,16 @@ export class IgxGridCellComponent implements OnInit, OnDestroy, AfterViewInit {
     }
 
     @HostListener('keydown.shift.tab', ['$event'])
+    public onShilftTabKey(event) {
+        if (this.isFirstCell) {
+            this.selectionApi.set_selection(this.cellSelectionID, []);
```

Comment:
Not sure this part is needed, the whole point on the fix is to allow the default browser handling to still work as before when [shift+]tab reaches the end of the row, right?
As far as I understand this is clearing the selection as you blur the cell, but that's kind of how the grid handles the selection currently - it sticks behind, even the active editing (see [Grid Editing](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid_editing.html#grid-editing)), so I don't think the keyboard navigation should be doing anything differently.

The default tab navigation will trigger blur just fine and the cell generally can decide if selection should be left behind.
@rkaraivanov @zdrawku Thoughts?